woocommerce/woocommerce-android
# Add toolbars to refund screens

Part of #10570 

### Description

This PR introduces individual toolbars for refund screens, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail for a paid order and click on Issue Refund CTA. Ensure the toolbar and its action work as expected on both phone and tablet.
